src/scripts/main.js

```diff
@@ -358,3 +358,23 @@ const people = [
 console.log(people); // you can remove it
```

Comment:
As a best practice, it's recommended to remove `console.log` statements when you're done with debugging. These statements can clutter the console output in a production environment.